src/internal/kopia/base_finder.go

```diff
@@ -238,6 +238,8 @@ func (b *baseFinder) findBasesInSet(
 			}
 
 		default:
+			// Preview backups automatically fall into this group so there's no need
+			// to add a case to skip them.
```

Comment:
I'd feel a little more comfortable with an explicit fallthrough as opposed to a comment.  The clarity may help maintenance/bug avoidance down the road.
```suggestion
                case model.PreviewBackup:
                    fallthrough
		default:
			// Preview backups automatically fall into this group so there's no need
			// to add a case to skip them.
```